packages/rocketchat-federation/server/federatedResources/FederatedMessage.js

```diff
@@ -0,0 +1,265 @@
+import { Meteor } from 'meteor/meteor';
+import { sendMessage, updateMessage } from 'meteor/rocketchat:lib';
+import { Messages, Rooms, Users } from 'meteor/rocketchat:models';
+
+import FederatedResource from './FederatedResource';
+import FederatedRoom from './FederatedRoom';
+import FederatedUser from './FederatedUser';
+
+import { FileUpload } from 'meteor/rocketchat:file-upload';
+import { Federation } from 'meteor/rocketchat:federation';
```

Comment:
you should not `import` from the package itself (I didn't know this is possible). you should should import as a file

Reply:
Oh, I see. That's fine, I can change this.